### DISCUSSION TITLE
push tag error

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#1675] [Q&A] [answered]
hi sir @r-devops ,iam not able to push the tags,
tags are getting created but when iam trying to push the tags , im not able to do it
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/9047b14e-c22c-432a-8f6a-3d6fc1b09d05)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/a5af31c4-5392-42b6-8342-ac6bcd5d0502)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/386d35b8-d0e8-4637-8044-98f2959efa28)
my workstation ip 54.84.89.66
please help me with this error sir

**Comment by sreedharm07** [comment, marked as answer]
@nandini965 
check mart this, it will push tags
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/f7f16e6b-39b3-480a-b54e-5715b5326e10)

**Reply by nandini965** [reply]
yes i selected pushed tags, but didnot find any tags in the jenkins
